response_operations_ui/templates/messages.html

```diff
@@ -1,45 +1,84 @@
 {% extends "layouts/base.html" %}
-{% set page_title = "Messages" %}
+{% from "components/search/_macro.njk" import onsSearch %}
 
+{% set page_title = "Messages" %}
+{% set hide_breadcrumbs = true %}
```

Comment:
Why are we hiding the breadcrumbs for this?  I've not fired up the code or looked at the design so it might be obvious but it seems wrong to do so as they're meant to be a navigation aid

Reply:
Turned off by Matt Simon as part of using design system , breadrumbs implemented via design system